readme-developer.md

```diff
@@ -116,22 +116,9 @@ Refer to the build pipeline files under `.github\` to tailor the build process t
 
 ### Running the Game
 
-For Endless Sky to run on Windows, it needs various .dll files included in the base directory. Go to `C:\dev64\bin\`, and copy the files to your base directory. These should include:
-
-  - glew32.dll
-  - **libgcc_s_seh-1.dll**
-  - libjpeg-62.dll
-  - libmad-0.dll
-  - libpng15-15.dll
-  - **libstdc++-6.dll**
-  - libturbojpeg.dll
-  - libwinpthread-1.dll
-  - OpenAL32.dll
-  - SDL2.dll
-  - soft_oal.dll
-  - zlib1.dll
-
-**After** you have copied all those libraries, take note of the **bolded** libraries. Those will need to be included from your MinGW installation, instead of from the precompiled ones. By default, you can go to `C:\Program Files\mingw-w64\x86_64-8.1.0-posix-seh-rt_v6-rev0\mingw64\bin`, and copy those libraries to your Endless Sky directory to overwrite the current ones.
+For Endless Sky to run on Windows, it needs various .dll files included in the base directory. Go to `C:\dev64\bin\`, and copy every file to your base directory.
+
+**After** you have copied all those libraries, you will need additional libraries from your MinGW installation: libstdc++-6.dll, libwinpthread-1.dll, libgcc_s_seh-1.dll. By default, you can go to `C:\Program Files\mingw-w64\x86_64-8.1.0-posix-seh-rt_v6-rev0\mingw64\bin`, and copy those libraries to your Endless Sky directory to overwrite the current ones.
```

Comment:
You sure these three are necessary? I'm able to launch without them.

Reply:
They're probably in your PATH, but to be safe they should get copied (it's mostly to protect against the case if incompatible libraries with the same name are in your PATH because of some other program; this happened once to Terin).

Reply:
Fair enough.